docker-compose.yml

```diff
@@ -175,28 +175,18 @@ services:
       - WORKERQUEUE=geotiff-validator
 
   geoserver:
-    image: meggsimum/geoserver:2.21.0
+    image: geoserver-docker.osgeo.org/geoserver:2.21.1
     restart: unless-stopped
     hostname: ${GEOSERVER_HOSTNAME}
     ports:
      - 8080:8080
     volumes:
       - ./geoserver_data:/opt/geoserver_data/:Z
     environment:
+      - INSTALL_EXTENSIONS=true
+      - STABLE_EXTENSIONS=wps
       - EXTRA_JAVA_OPTS=-Xms1g -Xmx2g
 
-  geoserver-init:
```

Comment:
Why is the init container removed?

Reply:
because there is currently no need for it. we just need to change the password once, everything else is covered by the geoserver image itself.

Reply:
OK, thanks for clarification

Reply:
:+1: